src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Row, Col, Strip } from "@canonical/react-components";
+
+export type Props = {
+  copyrightYear: number;
+  companyName: string;
+};
+
+const Footer: React.FC<Props> = ({ copyrightYear, companyName }) => {
```

Comment:
I don't think `companyName` will need to be a variable as it will always be "Canonical Ltd."

Reply:
I'll remove it :)